src/components/Menus/MenuDivider.js

```diff
@@ -0,0 +1,26 @@
+import React      from 'react'
+import PropTypes  from 'prop-types'
+import { colors } from '../../styles'
+
+const styles = {
+  backgroundColor: colors.GRAY_88,
+  width: '100%',
+  margin: '8px 0',
+  height: 1,
+  border: 'none'
+}
+
+const MenuDivider = (props) => {
+  const { style } = props
+
+  return (
+    <hr style={Object.assign(styles, style)}/>
```

Comment:
This will change the `styles` for all future `MenuDividers` since it will mutate the constant above. I would prefer the syntax `{... styles, ... style}` in the future to avoid this issue.